sessionctx/variable/session.go

```diff
@@ -574,6 +575,8 @@ type SessionVars struct {
 	SysWarningCount int
 	// SysErrorCount is the system variable "error_count", because it is on the hot path, so we extract it from the systems
 	SysErrorCount uint16
+	// generalPlanCacheStmts stores PlanCacheStmts for general plan cache.
+	generalPlanCacheStmts *kvcache.SimpleLRUCache
```

Comment:
Should we initialize this in the `NewSessionVars()` function.

Reply:
Lazy initialization seems a better choice.